packages/patternfly-4/react-core/src/components/Switch/Switch.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Switch/switch.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** id for the label. */
+  id: PropTypes.string.isRequired,
```

Comment:
since this id is only used to glue the label and the input together for accessibility purposes, i think we can generate it instead and make it optional. Example here:
https://github.com/patternfly/patternfly-react/blob/a331add0ca90f5d3feef9c4752651c957535f219/packages/patternfly-4/react-core/src/components/Nav/NavGroup.js#L27